Documentation/install/upgrade.rst

```diff
@@ -31,16 +31,14 @@ image. In order to reduce the downtime of the agent, the new image version can
 be pre-pulled. It also verifies that the new image version can be pulled and
 avoids ErrImagePull errors during the rollout.
 
-.. code:: bash
+.. parsed-literal::
 
-    helm template cilium \
-      --namespace=kube-system \
-      --set preflight.enabled=true \
-      --set agent.enabled=false \
-      --set config.enabled=false \
-      --set operator.enabled=false \
-      > cilium-preflight.yaml
-    kubectl create cilium-preflight.yaml
+    helm install cilium-preflight |CHART_RELEASE| \\
```

Comment:
Ah, interesting, there's also an assumption here that there's separate `cilium-preflight` charts. Note that previously we generated the template from Cilium.

For this I think we'll need to either add those separate charts, or switch this back to `cilium`:

```suggestion
    helm install cilium |CHART_RELEASE| \\
```

Reply:
@joestringer This section will render as `helm install cilium-preflight cilium/cilium --version 1.7.0`. This command will create another cilium deployment named `cilium-preflight` with enabled `preflight`, you can deploy multiple releases of the same chart to a cluster. This deployment can be removed using `cilium-preflight` name once image pull is verified. If I'm not mistaken these changes will have the same logic as before but use only `helm` CLI.

Reply:
Ah, my mistake, thanks for clarifying.